src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int INDEX_OF_NAME_OF_PRODUCT = 0;
+    private static final int INDEX_OF_COST_OF_PRODUCT = 1;
+    private static final String CSV_DIVIDER = ",";
+    private static final String BUY_ITEM = "buy";
+
+    public static void getStatistic(String fromFileName, String toFileName) {
+        generateReport(fromFileName, toFileName);
+    }
+
+    private static List<String> readFile(String fromFileName) {
+        File fileForRead = new File(fromFileName);
+        List<String> listWithInfoAboutProducts;
+        try {
+            listWithInfoAboutProducts = Files.readAllLines(fileForRead.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from the file " + fileForRead, e);
+        }
+        return listWithInfoAboutProducts;
+    }
+
+    private static void writeToFile(String fromFileName, String toFileName) {
+        File fileForWrite = new File(toFileName);
+        try {
+            fileForWrite.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't create new file" + fileForWrite, e);
+        }
```

Comment:
useless part of the program. Cause `BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))` creates a new file by default if it doesn't exist.
```suggestion
```